client/src/index.css

```diff
@@ -95,3 +95,17 @@ body {
   border-top: 1px solid #f5f5f5;
   padding: 10px;
 }
+.settings-btn {
```

Comment:
@taleymirza can we create a separate css file related to `comment section` and move all these changes into this one. 
This is because if we put all css in `index.css` then in future index.css will be messy.